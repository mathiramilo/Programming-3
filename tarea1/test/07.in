generar_grafo
100000 con_ciclo
generar_grafo
100000 sin_ciclo
Fin
