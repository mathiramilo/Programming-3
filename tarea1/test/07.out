1>Grafo generado de tamaño 100000
Hay un ciclo 
2>Grafo generado de tamaño 100000
Hay orden topologico 
3>Fin.
